packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx

```diff
@@ -134,77 +124,48 @@ function Branding({ application, isActive }: Props) {
                 <TextInput {...register('displayName')} placeholder={application.name} />
               </FormField>
             )}
-            {isUserAssetsServiceReady && (
-              <FormField title="application_details.branding.application_logo">
-                <LogoUploader isDarkModeEnabled={isDarkModeEnabled} />
-              </FormField>
-            )}
-            {/* Display the TextInput field if image upload service is not available */}
-            {!isUserAssetsServiceReady && (
-              <FormField title="application_details.branding.application_logo">
-                <TextInput
-                  {...register('branding.logoUrl', {
-                    validate: (value) =>
-                      !value || uriValidator(value) || t('errors.invalid_uri_format'),
-                  })}
-                  placeholder={t('sign_in_exp.branding.logo_image_url_placeholder')}
-                  error={errors.branding?.logoUrl?.message}
-                />
-              </FormField>
-            )}
-            {/* Display the Dark logo field only if the dark mode is enabled in the global sign-in-experience */}
-            {!isUserAssetsServiceReady && isDarkModeEnabled && (
-              <FormField title="application_details.branding.application_logo_dark">
-                <TextInput
-                  {...register('branding.darkLogoUrl', {
-                    validate: (value) =>
-                      !value || uriValidator(value) || t('errors.invalid_uri_format'),
-                  })}
-                  placeholder={t('sign_in_exp.branding.dark_logo_image_url_placeholder')}
-                  error={errors.branding?.darkLogoUrl?.message}
-                />
-              </FormField>
-            )}
+            <LogoAndFavicon
+              control={control}
+              register={register}
+              theme={Theme.Light}
+              type="app_logo"
+              logo={{ name: 'branding.logoUrl', error: errors.branding?.logoUrl }}
+              favicon={{
+                name: 'branding.favicon',
+                error: errors.branding?.favicon,
+              }}
+            />
+            <LogoAndFavicon
+              control={control}
+              register={register}
+              theme={Theme.Dark}
+              type="app_logo"
+              logo={{ name: 'branding.darkLogoUrl', error: errors.branding?.darkLogoUrl }}
+              favicon={{
+                name: 'branding.darkFavicon',
+                error: errors.branding?.darkFavicon,
+              }}
+            />
```

Comment:
Should we check if the dark theme is enabled first? If not, then this should NOT be rendered IMO

Reply:
no, it's by design. let's discuss with designers if needed